spec/controllers/messages_controller_spec.rb

```diff
@@ -34,15 +34,15 @@ def valid_message_request
 
       it "redirects to the message show page" do
         request.session[:user_id] = user.id
-        valid_message_request
+        post :create, params: { message: { email: message.email, content: message.content } }
```

Comment:
Use `FactoryGirl` `attributes_for` to generate message params 👇  https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#using-factories